integration-test/step_definitions/support/common.js

```diff
@@ -0,0 +1,137 @@
+const axios = require("axios");
+const crypto = require("crypto");
+
+
+function post(url, headers, data) {
+    return axios.post(url, data, {headers})
+        .then(res => {
+            return res;
+        })
+        .catch(error => {
+            return error.response;
+        });
+}
+  
+function getAuthorizationTokenUsingMasterKey(verb, resourceType, resourceId, date, masterKey) {
+    var key = new Buffer(masterKey, "base64");
+
+    var text = (verb || "").toLowerCase() + "\n" +   
+               (resourceType || "").toLowerCase() + "\n" +   
+               (resourceId || "") + "\n" +   
+               date.toLowerCase() + "\n" +   
+               "" + "\n";  
+  
+    var body = new Buffer(text, "utf8");  
+    var signature = crypto.createHmac("sha256", key).update(body).digest("base64");  
+  
+    var MasterToken = "master";  
+  
+    var TokenVersion = "1.0";  
+  
+    return encodeURIComponent("type=" + MasterToken + "&ver=" + TokenVersion + "&sig=" + signature);  
+}
+
+function createSharedAccessToken(uri, saName, saKey) { 
+    if (!uri || !saName || !saKey) { 
+            throw "Missing required parameter"; 
+        } 
+    var encoded = encodeURIComponent(uri); 
+    var now = new Date(); 
+    var day = 60*60*24;
+    var ttl = Math.round(now.getTime() / 1000) + day;
+    var signature = encoded + '\n' + ttl; 
+    var hash = crypto.createHmac('sha256', saKey).update(signature, 'utf8').digest('base64'); 
+    return 'SharedAccessSignature sr=' + encoded + '&sig=' + encodeURIComponent(hash) + '&se=' + ttl + '&skn=' + saName; 
+}
+
+function randomId(length) {
+    var result = '';
+    var alphaNumeric = 'abcdefghijklmnopqrstuvwxyz0123456789';
+    var alphaNumericLength = alphaNumeric.length;
+    for (var i = 0; i < length; i++) {
+        result += alphaNumeric.charAt(Math.floor(Math.random() * alphaNumericLength));
+    }
+    return result;
+}
+
+function sleep(milliseconds) {
```

Comment:
could you use something like https://www.tutorialspoint.com/javascript-sleep-function?